scripts/pylib/twister/twisterlib/reports.py

```diff
@@ -584,35 +584,39 @@ def summary(self, results, ignore_unrecognized_sections, duration):
             pass_rate = 0
 
         logger.info(
-            "{}{} of {}{} test configurations passed ({:.2%}), {} built (not run), {}{}{} failed, {}{}{} errored, {} skipped with {}{}{} warnings in {:.2f} seconds".format(
-                Fore.RED if failed else Fore.GREEN,
-                results.passed,
-                results.total,
-                Fore.RESET,
-                pass_rate,
-                results.notrun,
-                Fore.RED if results.failed else Fore.RESET,
-                results.failed,
-                Fore.RESET,
-                Fore.RED if results.error else Fore.RESET,
-                results.error,
-                Fore.RESET,
-                results.skipped_configs,
-                Fore.YELLOW if self.plan.warnings else Fore.RESET,
-                self.plan.warnings,
-                Fore.RESET,
-                duration))
+            f"{Fore.RED if failed else Fore.GREEN}{results.passed}"
+            f" of {results.total - results.skipped_configs}{Fore.RESET}"
+            f" executed test configurations passed ({pass_rate:.2%}),"
+            f" {f'{Fore.RED}{results.notrun}{Fore.RESET}' if results.notrun else f'{results.notrun}'} built (not run),"
```

Comment:
```suggestion
            f" {f'{TwisterStatus.get_color(TwisterStatus.NOTRUN}{results.notrun}{Fore.RESET}' if results.notrun else f'{results.notrun}'} built (not run),"
```
and in other places ?

Reply:
I don't see it in other places, at least those changed by this PR.